futures-util/src/async_await/join.rs

```diff
@@ -47,3 +47,64 @@ macro_rules! join {
         )*)
     } }
 }
+
+/// Attempts to poll multiple futures simultaneously, returning a `Result`
+/// of a tuple of all of the results once complete.
```

Comment:
``Polls multiple futures simultaneously, resolves to a [`Result`] containing a tuple of all success values.``

(The tuple does not contain results)